src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarShippingInformation.tsx

```diff
@@ -1,28 +1,30 @@
+import { ActionType, type ClickedOnLearnMore } from "@artsy/cohesion"
 import { Spacer, Text } from "@artsy/palette"
+import { ArtsyShippingEstimate } from "Components/ArtsyShippingEstimate"
+import { RouterLink } from "System/Components/RouterLink"
 import type { ArtworkSidebarShippingInformation_artwork$data } from "__generated__/ArtworkSidebarShippingInformation_artwork.graphql"
 import { createFragmentContainer, graphql } from "react-relay"
-
-import { ActionType, type ClickedOnLearnMore } from "@artsy/cohesion"
-import { RouterLink } from "System/Components/RouterLink"
 import { useTracking } from "react-tracking"
 
 export interface ShippingInformationProps {
   artwork: ArtworkSidebarShippingInformation_artwork$data
 }
+const CALCULATED_IN_CHECKOUT_MESSAGE = "Shipping: Calculated in checkout"
 
 const ArtworkSidebarShippingInformation: React.FC<
   React.PropsWithChildren<ShippingInformationProps>
-> = ({
-  artwork: {
+> = ({ artwork }) => {
+  const {
     isUnlisted,
     priceIncludesTaxDisplay,
     shippingOrigin,
     shippingInfo,
     pickupAvailable,
     taxInfo,
-  },
-}) => {
+  } = artwork
   const { trackEvent } = useTracking()
+  const globalArtsyShipping =
+    !!artwork.artsyShippingDomestic || !!artwork.artsyShippingInternational
```

Comment:
Don't think this is an accurate condition that we want here. Believe it has to be `!!artwork.artsyShippingDomestic && (!!artwork.artsyShippingInternational || (!artwork.artsyShippingInternationa && !artwork.international_shipping_fee))`. So either arta shipping both domestic and international or arta domestic and international not configured.

Reply:
Let's follow up on this in person, i believe you have the expertise and it's a small change if so